lib/mayaUsd/ufe/MayaUsdUIInfoHandler.cpp

```diff
@@ -0,0 +1,92 @@
+//
+// Copyright 2023 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#include "MayaUsdUIInfoHandler.h"
+
+#include <maya/MDoubleArray.h>
+#include <maya/MGlobal.h>
+
+namespace MAYAUSD_NS_DEF {
+namespace ufe {
+
+MayaUsdUIInfoHandler::MayaUsdUIInfoHandler()
+    : UsdUfe::UsdUIInfoHandler()
+{
+    // Register a callback to invalidate the invisible color.
+    fColorChangedCallbackId = MEventMessage::addEventCallback(
+        "DisplayRGBColorChanged", onColorChanged, reinterpret_cast<void*>(this));
```

Comment:
Split off Maya stuff into this derived class. Mostly to handle this Maya callback when the color prefs changed.